asv/plugins/regressions.py

```diff
@@ -207,8 +207,10 @@ def _save_feed(cls, conf, benchmarks, data, revisions, revision_to_hash):
                 # as the same one, as long as the benchmark name and
                 # commits match.
                 id_context = [name, revision_to_hash.get(rev1, ""), revision_to_hash.get(rev2, "")]
+                id_rev = rev1 if rev1 is not None else rev2
```

Comment:
Shouldn't id_rev be set, by default to `rev2`?
`rev2` seems to be the latest change, and in case of regression, it is the one "introducing" it.